autoPyTorch/pipeline/components/setup/network/base_network.py

```diff
@@ -21,6 +22,9 @@ def __init__(
         self.random_state = random_state
         self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu") if device is None else device
 
+        self._fit_requirements = [FitRequirement("input_shape", tuple),
+                                  FitRequirement("output_shape", tuple)]
```

Comment:
In the new PR #34 this will fail as the type argument has been changed to supported_types which expects an iterable.

Reply:
What type should it be then? I will change it

Reply:
just an Iterable over the type. So for example here it can be ```[FitRequirement("input_shape", [tuple, ]),
                                  FitRequirement("output_shape", [tuple, ])]```

Reply:
Fixed